source/Projectile.cpp

```diff
@@ -35,7 +35,7 @@ namespace {
 
 
 
-Projectile::Projectile(const Ship &parent, Point position, Angle angle, const Weapon *weapon)
+Projectile::Projectile(const Ship &parent, Point position, Angle angle, Angle projectileInaccuracy, const Weapon *weapon)
```

Comment:
I did this to retain information as I believe it's significant to know what changes the projectile is making from the heading given to it, but I'll look at closer, both in master here and the Submunition PR.

Reply:
No, it looks like the angle is changed before all of the math is done.  I'll fix it.  It can always be changed to this way if something in the constructor needs to know.